docs/projects/tcet-linux-website/frontend-components/download-section.mdx

```diff
@@ -0,0 +1,142 @@
+   5. Lastly in the `right column` we have added ***an image of a man with a laptop*** for beautification. 
+:::
+
+## Download Section & Gradient
+
+**Here's a breakdown of the code to understand it's functionality:**
+
+1. This code represents the start of the DownloadSection component. The div element with the id attribute `download` serves as a container for the entire section. It has classes for styling, such as a full width, background color, relative positioning, and hidden horizontal overflow.
+ 
+     ```jsx title="DownloadSection.astro" {1} showLineNumbers
+     <div  id="download"  class="w-full bg-slate-800 relative overflow-x-hidden px-10 lg:px-12">
+     // Furthur components were added 
+     </div>
+     ```
+
+2. This div element adds a `gradient background` to the section. It has the class `downloadSectionGradient` and the style `pointer-events-none` to prevent any interaction with the element.
+ 
+    ```  jsx title="DownloadSection.astro" {1} showLineNumbers
+    <div class="downloadSectionGradient pointer-events-none">
+    // Components are added 
+    </div>
+
+    ```
+
+## Left Column
+
+1. This div element displays the heading `Downloads` in white text with a bold font. The font size varies for different screen sizes.
+
+    ```   jsx title="DownloadSection.astro" {2} showLineNumbers
+       <div class="text-white font-semibold text-4xl lg:text-6xl mb-4 lg:mb-9">
+        Downloads
+      </div> 
+      // Multiple such headings can be added 
+
+    ```
+
+2. This p element contains a paragraph of text in `white color`. It provides information about the **availability of TCET Linux source code and installers for download**.
+
+  ``` jsx title="DownloadSection.astro" {2} showLineNumbers
+            <p class="text-white flex flex-wrap font-normal text-sm md:text-xl">
+          TCET Linux source code and installers are available for download <br /> for all versions!!
+        </p>
+
+  ```
+
+3. This **a tag** `link element` represents the `download button`. It has classes for styling, including a blue background color, padding, cursor style, rounded corners, and a border. The button contains the text ***Download*** and an image of a ***download icon***. The href attribute specifies the Github URL to downloadable ***iso image file*** which is linked to the [***Download***](https://github.com/tcet-opensource/tcet-linux/releases/download/v0.5-beta/tcetlinux-2023.05.05-x86_64.iso). It is represented by line **1-4**. The `button` element in this section represents a **clickable** button. It has styles applied as per our requirements.
+
+  ``` jsx title="DownloadSection.astro" {2} showLineNumbers
+          <a class="flex flex-row bg-blue-900 w-fit py-3 px-5 cursor-pointer gap-5 items-center justify-center rounded-full border-custom-w border-custom-border-color hover:bg-blue-500 font-semibold text-base" href="https://github.com/tcet-opensource/tcet-linux/releases/download/v0.5-beta/tcetlinux-2023.05.05-x86_64.iso" >
```

Comment:
Always give upper and lower references of the code via comments
Make this changes in all code blocks